### ISSUE TITLE
Content: Include `wc` in discussion of file inspection tools

### ISSUE THREAD

**Issue by jugglinmike** [bocoup-education/speaking-nix#4]
The section on "inspection" in the "File Management" chapter covers `grep` and `find`, but it would benefit from some discussion of the `wc` utility as well.

**Comment by gnarf** [comment]
You have `wc` under filesystem stuff before it, was there more you wanted to go into?

**Comment by jugglinmike** [comment]
Oh, you're right. It looks like I included a slide on `wc` in that chapter,
after all. I think that might be a little too much information for the audience
at that point, so I've moved it to the discussion on file management.

Limiting the available tools risks making this chapter's exercise a little too
boring. I haven't started writing the "Treasure Hunt" activity yet, but
depending on how that goes, I may move `wc` back.

Another thought about organization: it may be short-sighted to label tools like
"wc" and "sed" as "file inspection utilities" because they really operate on
streams. Describing them in terms of files may make the concepts more
approachable to students initially, but it might also increase the cognitive
leap they have to make when the time comes to start thinking about pipes.

In any case, the problem motivating this issue is resolved, so I'm going to
close it. Thanks for taking a look, @gnarf!
